website/docs/r/cloud_project_containerregistry_oidc.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory : "Managed Private Registry"
+---
+
+# ovh_cloud_project_containerregistry_oidc
+
+Creates an OIDC configuration in an OVHcloud Managed Private Registry.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_containerregistry_oidc" "my-oidc" {
+  service_name = "XXXXXX"
+  registry_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+
+  #required field
+  oidc_name          = "my-oidc-provider"
+  oidc_endpoint      = "https://xxxx.yyy.com"
+  oidc_client_id     = "xxx"
+  oidc_client_secret = "xxx"
+  oidc_scope         = "openid,profile,email,offline_access"
+
+  #optional field
+  oidc_groups_claim = "groups"
+  oidc_admin_group  = "harbor-admin"
+  oidc_verify_cert  = true
+  oidc_auto_onboard = true
+  oidc_user_claim   = "preferred_username"
+  delete_users      = false
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_name` - The ID of the public cloud project. If omitted, the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used. **Changing this value recreates the resource.**
+* `registry_id` - The ID of the Managed Private Registry. **Changing this value recreates the resource.**
+* `oidc_name` - The name of the OIDC provider.
+* `oidc_endpoint` - The URL of an OIDC-compliant server.
+* `oidc_client_id` - The client ID with which Harbor is registered as client application with the OIDC provider.
+* `oidc_client_secret` - The secret for the Harbor client application.
```

Comment:
Add a message for this field and `delete_users` in order to tell that these field can only be set during the creating

Reply:
This field can be modified, but not delete_users